yamllint/rules/line_length.py

```diff
@@ -140,7 +140,11 @@ def check(conf, line):
                 start += 1
 
             if start != line.end:
-                if line.buffer[start] in ('#', '-'):
+                if line.buffer[start] in ('#'):
```

Comment:
In Python, `in ('#')` is equal to `in '#'`, which works but doesn't make sense here.
Instead you can use `== '#'`.

Same for `in ('-')`.

Reply:
Done, please check